packages/shared/src/lib/utils.shared.test.ts

```
@@ -4,18 +4,22 @@
   isPositiveInteger,
+  isUndefined,
   isValidPort,
   makeUuid,
+  noop,
+  noopFalse,
   noopTrue,
-  omitUndefined,
   partition,
   pluralize,
   pluralizeWithCount,
+  safeAssertNever,
```

ESLint

[error] 'safeAssertNever' is defined but never used.

(unused-imports/no-unused-imports)